src/cp/cp.rs

```diff
@@ -46,7 +46,6 @@ use std::str::FromStr;
 use uucore::fs::{canonicalize, CanonicalizeMode};
 use walkdir::WalkDir;
 #[cfg(target_os = "linux")] use std::os::unix::io::IntoRawFd;
-use std::fs::File;
```

Comment:
I think you should just mark this as `#[cfg(target_os = "linux")]` instead.  Removing it causes the build to fail on Linux.

Reply:
Yeah, fixed, thanks